src/components/Comments/DetailDomainScreenCommentItem.js

```diff
@@ -205,7 +201,7 @@ const styles = StyleSheet.create({
   time: {
     fontFamily: fonts.inter[400],
     fontSize: 10,
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the timestamp has been updated. This change should be checked for consistency with other timestamps in the app and for its impact on readability.
```diff
- color: COLORS.blackgrey,
+ color: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->